Submissions/Laguerta_Week2.playground/Contents.swift

```diff
@@ -0,0 +1,71 @@
+//: Simple Values
+//: 1. Create a constant with an explicit type of Float that equals 4.
+let explicitFloat: Float = 4.0
+
+//: 2. Try removing the conversion to String from the last line. Copy the error and place it in a block comment below this block of code.
+let label = "The width is "
+let width = 94
+let widthLabel = label + String(width)
+
+/*
+ error: Laguerta_Week2.playground:2:24: error: binary operator '+' cannot be applied to operands of type 'String' and 'Int'
+ let widthLabel = label + width
+                  ~~~~~ ^ ~~~~~
+*/
+//: 3. Use \() to include a floating-point calculation in a string and to include someone's name in a greeting.
+var name = "Sandra"
+let ounces: Float = 1.1
+print("Hi, \(name)! Did you know that healthy parakeets should weigh between \(ounces) and \(ounces + 0.3) ounces? So cute! Very small!")
+
+//: Control Flow
+//: 1. Change optional Name to nil. What greeting do you get? Add an else clause that sets a different greeting if optionalName is nil.
+var optionalString: String? = "Hello"
+print(optionalString == nil)
+
+var optionalName: String? = nil
+var greeting = "Hello!"
+if let name = optionalName {
+    greeting = "Hello, \(name)"
+}else{
+    greeting = "Who are you, stranger?"
+}
+print(greeting)
+//When optionalName = nil and there is no added else clause, nothing is returned. Now, "Who are you, stranger?" will be returned if
```

Comment:
`greeting` has an initial value of "Hello!" -- without an else clause and with an `optionalName` set to nil, this value will not be altered. Your greeting will be "Hello!"